glean-core/android/src/main/java/mozilla/telemetry/glean/Glean.kt

```diff
@@ -363,12 +341,7 @@ open class GleanInternalAPI internal constructor () {
      * @param experimentId The id of the experiment to deactivate.
      */
     fun setExperimentInactive(experimentId: String) {
```

Comment:
These experiment API functions are razor thin now...  any reason to have the wrapper around the `gleanSetExperiment*` functions at all (aside from consistency with the other "glean" prefixed UDL functions)?

Reply:
Because API consistency. We need those exposed on the global Glean object. We can only create that Glean object on the Kotlin side, needing to wrap UDL-exported functions.

We _could_ have an `Glean` interface in the UDL which has those as methods, but we most likely still need to wrap that interface on the Kotlin side, because of the additional functionality that still exists.
Plain old functions seemed more reasonable to me.